frontend/server/fileSystem.js

```diff
@@ -49,7 +49,7 @@ const pipelineSpecBuilder = async (items, dir) => {
       const stat = await fs.stat(itemPath);
 
       if (stat.isDirectory()) {
-        const pipelineSpecFile = path.join(itemPath, `${item}.json`);
+        const pipelineSpecFile = path.join(itemPath, PIPELINE_SPECS_FILE_NAME);
```

Comment:
Is this backwards compatible with loading existing pipelines?

Reply:
No you're right it should be fine, as long as canny edge has the updated name it's not an issue for new versions.